doc/classes/Transform2D.xml

```diff
@@ -17,7 +19,7 @@
 		<constructor name="Transform2D">
 			<return type="Transform2D" />
 			<description>
-				Constructs a default-initialized [Transform2D] set to [constant IDENTITY].
+				Constructs a [Transform2D] identical to the [constant IDENTITY].
```

Comment:
```suggestion
				Constructs a [Transform2D] identical to [constant IDENTITY].
```
Just a suggestion, but IMO "identical to the IDENTITY" reads funny? Sounds like half of a sentence. "The identity of what?!" :) "to IDENTITY" makes it sounds like referring to a name of a thing, i.e., a constant, IMO?

Reply:
We do not speak about the IDENTITY.